mp4parse/src/tests.rs

```diff
@@ -388,6 +388,36 @@ fn read_vpcc() {
     assert!(r.is_ok());
 }
 
+// TODO: it'd be better to find a real sample here.
+#[test]
+fn read_vpcc_version_1() {
+    let data_length = 12u16;
+    let mut stream = make_fullbox(BoxSize::Auto, b"vpcC", 1, |s| {
+        s.B8(2)     // profile
+         .B8(0)     // level
+         .B8(0x82)  // bitdepth (4 bits), chroma (3 bits), video full range (1 bit)
+         .B8(0)     // color primaries
+         .B8(0)     // transfer characteristics
+         .B8(0)     // matrix
```

Comment:
The draft spec says, "If `matrixCoefficients` is 0 (RGB), the chroma sampling MUST be 3 (4:4:4)." So I think you need to set whatever the YCbCr value is for matrix, or set 0b1000_011_0 for the bitdepth/chroma/range field.